_data/chains/eip155-41500.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Opulent-X BETA",
+  "chainId": 41500,
+  "shortName": "ox-beta",
+  "chain": "Opulent-X",
+  "networkId": 41500,
+  "nativeCurrency": {
+    "name": "Oxyn Gas",
+    "symbol": "OXYN",
+    "decimals": 18
+  },
+  "rpc": ["https://connect.opulent-x.com"],
+  "faucets": [],
+  "infoURL": "https://beta.opulent-x.com",
+
```

Comment:
```suggestion
```

Reply:
copy. Thanks.

Reply:
I am done remove extra line but I got an error (Some checks were not successful)

Reply:
can you run the prettier?

Reply:
Done. It looks good. Thank you so much.

Reply:
Oh. I got still.

Reply:
do `npx prettier --write _data/*/*.json` then commit

Reply:
Done and committed.

Reply:
sure? I still see a diff after running the prettier:
```

--- a/_data/chains/eip155-18000.json
+++ b/_data/chains/eip155-18000.json
@@ -1,9 +1,7 @@
 {
   "name": "Frontier of Dreams Testnet",
   "chain": "Game Network",
-  "rpc": [
-    "https://rpc.fod.games/"
-  ],
+  "rpc": ["https://rpc.fod.games/"],
   "nativeCurrency": {
     "name": "ZKST",
     "symbol": "ZKST",
@@ -14,9 +12,11 @@
   "chainId": 18000,
   "networkId": 18000,
   "infoURL": "https://goexosphere.com",
-  "explorers": [{
-    "name": "Game Network",
-    "url": "https://explorer.fod.games",
-    "standard": "EIP3091"
-  }]
+  "explorers": [
+    {
+      "name": "Game Network",
+      "url": "https://explorer.fod.games",
+      "standard": "EIP3091"
+    }
+  ]
 }
```

Reply:
Opsssss sorry. I did, but I only committed my file.

Reply:
I have just ran prettier and committed all affected files.

Reply:
![image](https://user-images.githubusercontent.com/22004059/188752789-109800ed-faa6-4860-9da0-cc721c889244.png)

This is what I did.